packages/mantine/src/components/header/Header.tsx

```diff
@@ -78,6 +78,16 @@ const defaultProps: Partial<HeaderProps> = {
     wrap: 'nowrap',
 };
 
+const getSpacing = (variant: HeaderVariant) => {
+    switch (variant) {
+        case 'secondary':
+            return 'xxs';
+        case 'primary':
+        default:
+            return 'xs';
+    }
```

Comment:
if/else can be simpler in that case ?
```suggestion
   if(variant == 'primary') {
     return 'xs';
   }
   return 'xxs';
```